dll/win32/shell32/CDefView.cpp

```diff
@@ -1513,33 +1543,24 @@ HRESULT CDefView::OpenSelectedItems()
     if (!hMenu)
         return E_FAIL;
 
-    hResult = GetItemObject(SVGIO_SELECTION, IID_PPV_ARG(IContextMenu, &m_pCM));
+    CComPtr<IContextMenu> pCM;
+    hResult = GetItemObject(SVGIO_SELECTION, IID_PPV_ARG(IContextMenu, &pCM));
+    MenuCleanup _(pCM, hMenu);
```

Comment:
That is just you, because it's actually a variable name 😘
This is a convention used in python and googletest among others to denote that this is note referenced anywhere.
In this case, the object just needs to exist, so it can cleanup on destruction.